docs/source/tutorials/semantic_image_search.ipynb

```diff
@@ -0,0 +1,1222 @@
+      "\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[t-SNE] Iteration 50: error = 80.2364502, gradient norm = 0.0046372 (50 iterations in 0.683s)\n",
+      "[t-SNE] Iteration 100: error = 80.7005768, gradient norm = 0.0025561 (50 iterations in 0.518s)\n",
+      "[t-SNE] Iteration 150: error = 80.7050171, gradient norm = 0.0005539 (50 iterations in 0.484s)\n",
+      "[t-SNE] Iteration 200: error = 80.6982803, gradient norm = 0.0003416 (50 iterations in 0.557s)\n",
+      "[t-SNE] Iteration 250: error = 80.6925049, gradient norm = 0.0001926 (50 iterations in 0.546s)\n",
+      "[t-SNE] KL divergence after 250 iterations with early exaggeration: 80.692505\n",
+      "[t-SNE] Iteration 300: error = 2.1915989, gradient norm = 0.0011361 (50 iterations in 0.550s)\n",
+      "[t-SNE] Iteration 350: error = 1.8386334, gradient norm = 0.0004558 (50 iterations in 0.526s)\n",
+      "[t-SNE] Iteration 400: error = 1.6945410, gradient norm = 0.0002595 (50 iterations in 0.522s)\n",
+      "[t-SNE] Iteration 450: error = 1.6175762, gradient norm = 0.0001783 (50 iterations in 0.542s)\n",
+      "[t-SNE] Iteration 500: error = 1.5713907, gradient norm = 0.0001393 (50 iterations in 0.536s)\n",
+      "[t-SNE] Iteration 550: error = 1.5422311, gradient norm = 0.0001151 (50 iterations in 0.560s)\n",
+      "[t-SNE] Iteration 600: error = 1.5234932, gradient norm = 0.0001010 (50 iterations in 0.551s)\n",
+      "[t-SNE] Iteration 650: error = 1.5115008, gradient norm = 0.0000915 (50 iterations in 0.566s)\n",
+      "[t-SNE] Iteration 700: error = 1.5029023, gradient norm = 0.0000870 (50 iterations in 0.550s)\n",
+      "[t-SNE] Iteration 750: error = 1.4966835, gradient norm = 0.0000775 (50 iterations in 0.523s)\n",
+      "[t-SNE] Iteration 800: error = 1.4912493, gradient norm = 0.0000669 (50 iterations in 0.531s)\n",
+      "[t-SNE] Iteration 850: error = 1.4860034, gradient norm = 0.0000644 (50 iterations in 0.525s)\n",
+      "[t-SNE] Iteration 900: error = 1.4816489, gradient norm = 0.0000584 (50 iterations in 0.522s)\n",
+      "[t-SNE] Iteration 950: error = 1.4776893, gradient norm = 0.0000580 (50 iterations in 0.532s)\n",
+      "[t-SNE] Iteration 1000: error = 1.4744453, gradient norm = 0.0000542 (50 iterations in 0.506s)\n",
+      "[t-SNE] KL divergence after 1000 iterations: 1.474445\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": []
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/plain": []
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "789db642f43f4d8ca1a19a0c818362d8",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
```

Comment:
This image is not showing when I build the docs. You may need to screenshot it and add it as an image manually. You can see examples in the CVAT tutorial.